dev/ResourceHelper/ResourceAccessor.h

```diff
@@ -149,5 +149,12 @@ class ResourceAccessor sealed
 #define SR_TeachingTipNotificationWithoutAppName L"TeachingTipNotificationWithoutAppName"
 #define SR_TabViewAddButtonName L"TabViewAddButtonName"
 #define SR_TabViewAddButtonTooltip L"TabViewAddButtonTooltip"
+#define SR_NumberBoxErrorNumbersOnly L"NumberBoxErrorNumbersOnly"
+#define SR_NumberBoxErrorNumbersAndSymbols L"NumberBoxErrorNumbersAndSymbols"
+#define SR_NumberBoxErrorDivByZero L"NumberBoxErrorDivByZero"
+#define SR_NumberBoxErrorInvalidInput L"NumberBoxErrorInvalidInput"
+#define SR_NumberBoxErrorRange L"NumberBoxErrorRange"
+#define SR_NumberBoxUpSpinButtonName L"NumberBoxUpSpinButtonName"
```

Comment:
>define [](start = 1, length = 6)

I wonder if we should change all of these to static constexpr instead of #defines, they should be basically equivalent but with the added benefit of type checking.

Reply:
Maybe it wouldn't be appropriate for this change.

---
In reply to: [340860778](https://github.com/microsoft/microsoft-ui-xaml/pull/1507#discussion_r340860778) [](ancestors = 340860778)